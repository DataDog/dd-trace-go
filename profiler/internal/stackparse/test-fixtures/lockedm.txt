goroutine 7 [running, locked to thread]:
testing.tRunner.func1.1(0x12081c0, 0x1290cd0)
	/usr/local/Cellar/go/1.15.6/libexec/src/testing/testing.go:1072 +0x30d
testing.tRunner.func1(0xc000001b00)
	/usr/local/Cellar/go/1.15.6/libexec/src/testing/testing.go:1075 +0x41a
panic(0x12081c0, 0x1290cd0)
	/usr/local/Cellar/go/1.15.6/libexec/src/runtime/panic.go:969 +0x1b9
gopkg.in/DataDog/dd-trace-go.v1/profiler/internal/stackparse.TestParse_Example(0xc000001b00)
	/Users/felix.geisendoerfer/go/src/github.com/DataDog/dd-trace-go/profiler/internal/stackparse/stackparse_test.go:21 +0x45
testing.tRunner(0xc000001b00, 0x1263488)
	/usr/local/Cellar/go/1.15.6/libexec/src/testing/testing.go:1123 +0xef
created by testing.(*T).Run
	/usr/local/Cellar/go/1.15.6/libexec/src/testing/testing.go:1168 +0x2b3
