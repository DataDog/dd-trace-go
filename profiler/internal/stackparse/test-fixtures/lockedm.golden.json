{
  "Errors": null,
  "Goroutines": [
    {
      "ID": 7,
      "State": "running",
      "Wait": 0,
      "LockedToThread": true,
      "Stack": [
        {
          "Func": "testing.tRunner.func1.1",
          "File": "/usr/local/Cellar/go/1.15.6/libexec/src/testing/testing.go",
          "Line": 1072
        },
        {
          "Func": "testing.tRunner.func1",
          "File": "/usr/local/Cellar/go/1.15.6/libexec/src/testing/testing.go",
          "Line": 1075
        },
        {
          "Func": "panic",
          "File": "/usr/local/Cellar/go/1.15.6/libexec/src/runtime/panic.go",
          "Line": 969
        },
        {
          "Func": "gopkg.in/DataDog/dd-trace-go.v1/profiler/internal/stackparse.TestParse_Example",
          "File": "/Users/felix.geisendoerfer/go/src/github.com/DataDog/dd-trace-go/profiler/internal/stackparse/stackparse_test.go",
          "Line": 21
        },
        {
          "Func": "testing.tRunner",
          "File": "/usr/local/Cellar/go/1.15.6/libexec/src/testing/testing.go",
          "Line": 1123
        }
      ],
      "FramesElided": false,
      "CreatedBy": {
        "Func": "testing.(*T).Run",
        "File": "/usr/local/Cellar/go/1.15.6/libexec/src/testing/testing.go",
        "Line": 1168
      }
    }
  ]
}
